tachyon/zk/plonk/circuit/expressions/expression_unittest.cc

```diff
@@ -0,0 +1,162 @@
+#include "gtest/gtest.h"
+
+#include "tachyon/base/random.h"
+#include "tachyon/math/elliptic_curves/bn/bn254/fr.h"
+#include "tachyon/zk/plonk/circuit/expressions/expression_factory.h"
+
+namespace tachyon::zk {
+
+using Fr = math::bn254::Fr;
```

Comment:
Did you intentionally not call `Init()` since it does nothing?

Reply:
But for consistency, it's good to call `Init()`! I'll do that.